SIPS/sip-20.md

```diff
@@ -1,22 +1,23 @@
 ---
 sip: 20
-title: External data entry point
+title: Advanced external communication and events
```

Comment:
Feels like the name is not reflecting what this SIP really is anymore? Looking for advice to change it if possible. Updated with one of the ideas I had.